chacha20_poly1305/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "chacha20-poly1305"
+version = "0.1.0"
+license = "CC0-1.0"
+repository = "https://github.com/rust-bitcoin/rust-bitcoin/"
+description = "The ChaCha20 stream cipher and Poly1305 MAC based AEAD."
+categories = ["cryptography"]
+keywords = ["crypto", "encryption"]
+readme = "README.md"
+edition = "2021"
+rust-version = "1.63.0"
+exclude = ["tests", "contrib"]
+
+[features]
+default = ["std"]
+std = ["alloc"]
+alloc = []
+
+[dev-dependencies]
+hex = { package = "hex-conservative", version = "0.2.0", default-features = false, features = ["alloc"] }
+
+[package.metadata.docs.rs]
+all-features = true
+rustdoc-args = ["--cfg", "docsrs"]
```

Comment:
FTR having this general-purpose encryption crate inside `bitcoin` repository is a bit odd but I think it's fine for now.